src/pybind/ceph_argparse.py

```diff
@@ -999,6 +1001,9 @@ def validate(args, signature, flags=0, partial=False):
     if flags & FLAG_MGR:
         d['target'] = ('mgr','')
 
+    if flags & FLAG_POLL:
+        d['poll'] = True
```

Comment:
@tchaikov The problem was that `ceph.in` looks at `len(valid_dict)` (ie the number of parameters) to decide whether it's a `pg` command or not. See [here](https://github.com/ceph/ceph/blob/master/src/pybind/ceph_argparse.py#L1164). Setting `valid_dict['poll'] = False` (previously done here) broke that logic. The fix is to keep `valid_dict['poll']` unset, unless the command server-side expects it. This is OK because the length of `valid_dict` isn't used for the plugin. There's already a check that `'poll' in valid_dict` whenever we want to access it, so having it unset is not an issue.

Reply:
ahh, thanks for the explanation! actually, i was debugging this yesterday.